.github/workflows/curldemo.yaml

```diff
@@ -0,0 +1,29 @@
+name: curldemo
+
+on:
+  push:
+    paths:
+      - 'deploy/test-apps/curldemo/**'
+
+jobs:
+  curldemo:
+    runs-on: ubuntu-20.04
+    env:
+      DOCKER_CLI_EXPERIMENTAL: "enabled"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Login to Dockerhub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: ./deploy/test-apps/curldemo/
+          file: ./Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKER_USER }}/k8gb-demo-curl:$GITHUB_RUN_NUMBER
```

Comment:
Let's stick with `latest` as we agreed in https://github.com/k8gb-io/k8gb/pull/709#discussion_r740856251